docs/en/latest/how-to-build.md

```diff
@@ -21,351 +25,220 @@ title: Installing Apache APISIX
-
-  The following is the solution of macOS, which is similar to that of other operating systems:
-
-    1. Install `openldap` with `brew install openldap`;
-    2. Locate installation directory with `brew --prefix openldap`;
-    3. Add the path to the project configuration file(choose one of the following two methods):
-       1. Solution A: You can set `LDAP_DIR` with `luarocks config` manually, for example `luarocks config variables.LDAP_DIR /opt/homebrew/cellar/openldap/2.6.1`;
-       2. Solution B: Of course, you can also choose to change the default configuration file of luarocks directly, execute the 'cat ~/.luarocks/config-5.1.lua' command, and then add the installation directory of 'openldap' to the file;
-       3. Example as follows:
-          variables = {
-              LDAP_DIR = "/opt/homebrew/cellar/openldap/2.6.1",
-              LDAP_INCDIR = "/opt/homebrew/cellar/openldap/2.6.1/include",
-          }
-
-     `/opt/homebrew/cellar/openldap/` is default path to install openldap on macOS(Apple Silicon) using brew.
-     `/usr/local/opt/openldap/` is default path to install openldap on macOS(Intel) using brew.
-
-6. To uninstall the Apache APISIX runtime, run:
-
-   ```shell
-   # Purge dependencies
-   make undeps
-   ```
-
-   **Note**: This operation will remove the files completely.
-
-#### LTS version installation via Source Release Package
-
-The [current LTS version](https://apisix.apache.org/downloads/) of Apache APISIX is `2.13.0`.
+```shell
+helm repo add apisix https://charts.apiseven.com
+helm repo update
+helm install apisix apisix/apisix --create-namespace  --namespace apisix
+```
 
-To install this version, set `APISIX_VERSION` in [Installation via Source Release Package](#installation-via-source-release-package) to this version and continue with the other steps.
+You can find other Helm charts on the [apisix-helm-chart](https://github.com/apache/apisix-helm-chart) repository.
 
-## Step 2: Install etcd
+</TabItem>
 
-This step is required only if you haven't installed [etcd](https://github.com/etcd-io/etcd).
+<TabItem value="rpm">
 
-Run the command below to install etcd via the binary in Linux:
+This installation method is suitable for CentOS 7 and Centos 8.
 
-```shell
-ETCD_VERSION='3.4.18'
-wget https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz
-tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
-  cd etcd-v${ETCD_VERSION}-linux-amd64 && \
-  sudo cp -a etcd etcdctl /usr/bin/
-nohup etcd >/tmp/etcd.log 2>&1 &
-```
+### Installation via RPM repository
```

Comment:
We should prompt the user to install etcd here instead of putting the etcd installation after the APISIX installation. We can hint here:
If installing via RPM, you need to install [etcd](#Installing-etcd) first.

Reply:
First install etcd and then install via RPM is recommended? I could add a link to the install etcd section. Since etcd installation can also be common for other installation methods as well.

Reply:
Yes, installing APISIX via RPM needs to install etcd first, otherwise, APISIX will not start. you can add a link.

Reply:
I think adding a "Prerequisite" section before "Installing APISIX"section, in which you'd
1. Mention about etcd as a prereq.
2. Add command line(s) for users to verify if they do have ectd now.
3. Add links and other stuff to it. 

This can solve all problems mentioned above.